plugins/UM3NetworkPrinting/src/Cloud/CloudApiClient.py

```diff
@@ -30,6 +31,14 @@
 """The generic type variable used to document the methods below."""
 
 
+try:
```

Comment:
I think it would be slightly better to do this in the same way as it's done for the xml_material_profile

Right now this will load the file when the cloudapi.py is imported, instead of first time that the data is needed.